plugins/css-blank-pseudo/src/browser.js

```diff
@@ -30,8 +33,27 @@ function generateHandler(replaceWith) {
 
 	if (replaceWith[0] === '.') {
 		selector = replaceWith.slice(1);
-		remove = (el) => el.classList.remove(selector);
```

Comment:
🤔 I actually do not mind `classList` here.

The default is to use attributes which works everywhere.
Users that absolutely want a class here can add their own `classList` polyfill.

Reply:
Maybe we should explain this were we document the option to provide a class?

Reply:
ok then!